images/win/scripts/ImageHelpers/TestsHelpers.ps1

```diff
@@ -53,6 +53,10 @@ function Invoke-PesterTests {
     if ($TestName) {
         $configuration.Filter.FullName = $TestName
     }
+    if ($TestFile -eq "*") {
+        $configuration.TestResult.Enabled = $true
+        $configuration.TestResult.OutputPath = "C:\image\Tests\testResults.xml"
+    }
```

Comment:
Could you please double check that Pester will print test output (test results) to console with this option?
I would prefer to see test results in build logs too